webpack/configs/prod.ts

```diff
@@ -0,0 +1,34 @@
+import * as UglifyJsPlugin from "uglifyjs-webpack-plugin";
+import * as webpack from "webpack";
+import * as merge from "webpack-merge";
+
+import CommonConfig from "./base";
+
+const ProdConfig: webpack.Configuration = {
+
+    mode: "production",
+
+    output: {
+        filename: "clarity.min.js"
+    },
+
+    plugins: [
+        new webpack.DefinePlugin({
+            "process.env.NODE_ENV": JSON.stringify("production")
+        })
+    ],
+
+    optimization: {
+        minimizer: [
+            new UglifyJsPlugin({
+                cache: false
+            })
+        ]
+    },
+
+    stats: {
+        warnings: false
```

Comment:
Why turn of warnings?

Reply:
We don't care about build warnings in production

---
In reply to: [250239240](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250239240) [](ancestors = 250239240)